File: C/practicals/Practical-3.c/p7.c
#include<stdio.h>
void main(){
		float CP=1200;
		float SP=1000;
		

		if (SP>CP){
			
			printf("\nAs Selling prise is grater than Cost Price\n");
			printf("profit is : %f\n",SP-CP);
		}else if(CP>SP){
			
			printf("\nAs Cost prise is grater than Selling Price\n");
			printf("Loss is : %f\n",CP-SP);
		}
}

File: JAVA/AsgnPract/4/p7.java
import java.util.*;

class seven{

	public static void main(String [] args){

		Scanner sc = new Scanner(System.in);

		System.out.print("Enter the Cost price: ");

		float cp = sc.nextFloat();

		System.out.print("Enter the Selling price: ");

		float sp = sc.nextFloat();

		if (sp > cp){

			System.out.println("PROFIT: "+ (sp - cp));
	
		}else if (cp > sp){
	
			System.out.println("LOSS: "+(cp - sp));
	
		}else{
			System.out.println("NO PROFIT NO LOSS");
		}


	}
}
